flake.nix

```diff
@@ -401,8 +401,12 @@
               export WEBKIT_DISABLE_COMPOSITING_MODE=1
               export XDG_DATA_DIRS=${pkgs.gsettings-desktop-schemas}/share/gsettings-schemas/${pkgs.gsettings-desktop-schemas.name}:${pkgs.gtk3}/share/gsettings-schemas/${pkgs.gtk3.name}:$XDG_DATA_DIRS
               export GIO_MODULE_DIR="${pkgs.glib-networking}/lib/gio/modules/";
-            '';
+            '' 
+            + (if pkgs.stdenv.isDarwin then ''
+              export PATH=''${PATH//'${pkgs.xcbuild.xcrun}/bin:'/}
```

Comment:
this is pretty hacky, normally the path handling is all done by the packages themselves

i understand that there's an upstream issue and that this is a workaround, so

- that should be very clearly documented in the code, with a link to the upstream discussion and workaround
- there should be a test showing the PATH modifications have added the necessary bin without breaking anything else

otherwise there's no way for anyone else to maiintain this code moving forward